### ISSUE TITLE
about fine-tune on MIT1003

### ISSUE THREAD

**Issue by ArchWing** [marcellacornia/mlnet#14]
Hi @marcellacornia 
Can you tell me more details about fine tuning on MIT1003?Such as learning rate config etc...
Thank you.

**Comment by marcellacornia** [comment]
Hi @ArchWing,
starting from the model trained on SALICON, we finetuned the network using 900 randomly selected training images and 103 validation images. The network configuration and the other parameters are the same used to train the network on the SALICON dataset. 

Since images from MIT1003 dataset have different sizes, we decided to zero-pad them to fit a 4 : 3 aspect ratio.

**Comment by ArchWing** [comment]
@marcellacornia 
Thank you very much!
Yeah, I see the size config is 640*480, but because of the limit of gpu capacity, I change it to 426 * 320.
